build/scripts/build-client-packages

```diff
@@ -188,7 +196,7 @@ install_deps() {
         exit 1
     fi
     if [ "$OS" = "rpm" ]; then
```

Comment:
Why it's called `$OS`? Maybe `PackageManager` or `PM`?

Reply:
> Why it's called `$OS`? Maybe `PackageManager` or `PM`?

`OS=deb` is fine, but `OS=rpm` definitely needs to be renamed to something like `rhel`.

Reply:
I've replaced OS with PACKAGE_MANAGER, which seems more reasonable.